offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/SelectItemsView.groovy

```diff
@@ -224,7 +225,12 @@ class SelectItemsView extends VerticalLayout implements Resettable{
         this.applyProjectManagement = new Button("Apply", VaadinIcons.PLUS)
         applyProjectManagement.setEnabled(false)
 
-        HorizontalLayout buttonLayout = new HorizontalLayout(previous,next)
+        this.removeItemsButton = new Button("Remove Selected Item", VaadinIcons.MINUS)
+        removeItemsButton.setEnabled(false)
+        removeItemsButton.setStyleName(ValoTheme.BUTTON_DANGER)
+
+        HorizontalLayout buttonLayout = new HorizontalLayout(previous, removeItemsButton, next)
+        buttonLayout.setComponentAlignment(removeItemsButton, Alignment.BOTTOM_CENTER)
```

Comment:
how about positioning the button on the upper left part of the grid (above the grid)?

Reply:
Yeah, i was thinking of alternatives as well and came to the conclusion, to wait for user feedback :)

Reply:
So you would keep it as is?

Reply:
that was my idea yes